src/server/transaction.h

```diff
@@ -349,6 +354,7 @@ class Transaction {
 
   // Stores the full undivided command.
   CmdArgList cmd_with_full_args_;
+  bool no_auto_enable_auto_journal_ = false;
```

Comment:
I would just call this `renabled_auto_journal_` and add a comment like `true if NO_AUTOJOURNAL command asked to enable auto journal`